styles.css

```diff
@@ -0,0 +1,922 @@
+    left: 1rem;
+  }
+
+  #b-icon {
+    color: #f42;
+    font-size: 1rem;
+    padding-right: 1rem;
+  }
+
+  #s-icon {
+    position: relative;
+    left: 5rem;
+    font-size: 1rem;
+    border-left: 1px solid gray;
+    border-right: 1px solid gray;
+    padding: 0.5rem 1rem 0.5rem 1rem;
+  }
+
+  .nav-below {
+    position: fixed;
+    width: 100%;
+    height: 5rem;
+    display: flex;
+  }
+
+  .another p {
+    text-align: left;
+    font-size: 0.8rem;
+    font-weight: bold;
+    padding-left: 0.5rem;
+    padding-right: 0.5rem;
+    width: 100%;
+  }
+
+  #nav-b p {
+    padding: 35px 0 0 0;
+    justify-content: flex-start;
+    align-items: center;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    overflow: hidden;
+  }
+
+  .main-news {
+    width: 100%;
+    grid-template-areas:
+      'top-side'
+      'middle-side'
+      'bottom-side';
+    grid-template-columns: 0.96fr;
+    grid-template-rows: auto;
+    gap: 0.5rem;
+  }
+
+  .big-picture {
+    grid-area: top-side;
+    width: 100%;
+    height: 20rem;
+    background-repeat: no-repeat;
+    background-position: top center;
+    background-size: cover;
+  }
+
+  .a-top {
+    grid-area: middle-side;
+  }
+
+  .b-bottom {
+    grid-area: bottom-side;
+  }
+
+  .side-pic {
+    width: 100%;
+  }
+
+  .logo-big-pic {
+    display: none;
+    overflow: hidden;
+  }
+
+  .title-details {
+    font-size: 0.5rem;
+  }
+
+  .title-details h2 {
+    font-size: 2rem;
+  }
+
+  footer {
+    width: 20.5rem;
+    display: grid;
+    grid-template-columns: 1.5fr;
+  }
+
+  .footer-bottom {
+    width: 100%;
+    padding: 1rem 2rem;
+  }
+
+  .accessible {
+    position: fixed;
+    bottom: 0;
+    right: 15px;
+    width: 1.5rem;
+    height: 2rem;
+    border-radius: 0;
+    font-size: 1rem;
+    padding: 0.4rem 0.3rem;
+  }
+
+  .bottom-icons {
+    padding: 0.7rem 0;
+  }
+
+  .footer-top ul li {
+    padding-left: 0.8rem;
+  }
+
+  .top-icons li a {
+    font-size: 0.8rem;
+    margin-bottom: 1rem;
+    background-color: gray;
+    padding: 5px;
+    border-radius: 50%;
+  }
+
+  .funds-section {
+    width: 98%;
+  }
+
+  .other-tech {
+    width: 100%;
+    margin: 2rem 0;
+    grid-template-columns: repeat(1, minmax(18.5rem, 1fr));
+    grid-template-rows: auto;
+    grid-row-gap: 1.1rem;
+  }
+
+  .single {
+    width: 95%;
+  }
+
+  .first-picture {
+    width: 100%;
+    height: 15rem;
+    background-image: linear-gradient(to bottom, rgba(245, 246, 252, 0.52), rgba(31, 22, 29, 0.73)), url(images/bitcoin.jpeg);
+    background-repeat: no-repeat;
+    background-position: bottom center;
+    background-size: cover;
+    font-size: 0.8rem;
+  }
+
+  .mid-section {
+    max-width: 100%;
+    width: 25rem;
+    height: 5rem;
+    flex-direction: row-reverse;
+    background-color: rgba(255, 239, 237, 0.55);
+    margin-bottom: 0.4rem;
+  }
+
+  .another {
+    text-align: left;
+    font-size: 0.8rem;
+    font-weight: bold;
+    max-width: 100%;
+    width: 18rem;

```

Comment:
maybe the problem with this section comes from here, try to modify the width to 100% and align the text to the right